screenpy/resolutions/custom_matchers/is_in_bounds.py

```diff
@@ -32,6 +41,9 @@ def __init__(
         upper_comparator: InequalityFunc,
         majorant: float,
     ) -> None:
+        if not minorant < majorant:
```

Comment:
This is a tiny gripe, but testing the negative when we don't need to always reads weird to me, like i have to do a double-take to understand it. Can we change this to:
```python
if minorant > majorant
```
or
```python
if majorant < minorant
```

Reply:
Technically it would be `if minorant >= majorant:` but yeah, easily changed.